osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -202,6 +203,9 @@ public override void OnEntering(ScreenTransitionEvent e)
                 dialogOverlay?.Push(new StorageErrorDialog(osuStorage, osuStorage.Error));
         }
 
+        [CanBeNull]
+        private Drawable proxiedLogo;
```

Comment:
Storing this is required because if you exit _very_ fast, you can somehow hit `LogoExiting()` _without_ a prior `LogoArriving()`, thus dying at

```
Unhandled exception. System.InvalidOperationException: No usage to return
   at osu.Game.Screens.Menu.OsuLogo.ReturnProxy() in /home/dachb/Documents/opensource/osu/osu.Game/Screens/Menu/OsuLogo.cs:line 461
   at osu.Game.Screens.Menu.MainMenu.LogoExiting(OsuLogo logo) in /home/dachb/Documents/opensource/osu/osu.Game/Screens/Menu/MainMenu.cs:line 271
   at osu.Game.Screens.OsuScreen.<onExitingLogo>b__80_0() in /home/dachb/Documents/opensource/osu/osu.Game/Screens/OsuScreen.cs:line 265
   at osu.Game.Screens.Menu.OsuLogo.<>c__DisplayClass39_0.<AppendAnimatingAction>g__runnableAction|0() in /home/dachb/Documents/opensource/osu/osu.Game/Screens/Menu/OsuLogo.cs:line 256
   at osu.Game.Screens.Menu.OsuLogo.AppendAnimatingAction(Action action, Boolean waitForPrevious) in /home/dachb/Documents/opensource/osu/osu.Game/Screens/Menu/OsuLogo.cs:line 261
   at osu.Game.Screens.OsuScreen.onExitingLogo() in /home/dachb/Documents/opensource/osu/osu.Game/Screens/OsuScreen.cs:line 265
   at osu.Game.Screens.OsuScreen.OnExiting(ScreenExitEvent e) in /home/dachb/Documents/opensource/osu/osu.Game/Screens/OsuScreen.cs:line 230
   at osu.Game.Screens.Menu.MainMenu.OnExiting(ScreenExitEvent e) in /home/dachb/Documents/opensource/osu/osu.Game/Screens/Menu/MainMenu.cs:line 334
   at osu.Framework.Screens.ScreenStack.exitFrom(IScreen source, Boolean shouldFireExitEvent, Boolean shouldFireResumeEvent, IScreen destination)
   at osu.Framework.Screens.ScreenStack.Exit(IScreen source)
   at osu.Framework.Screens.ScreenExtensions.Exit(IScreen screen)
   at osu.Game.Screens.Menu.MainMenu.<OnExiting>b__62_0() in /home/dachb/Documents/opensource/osu/osu.Game/Screens/Menu/MainMenu.cs:line 318
```

Alternative would be making `ReturnProxy()` safely re-entrant I guess.